storm-core/src/saveVersion.sh

```diff
@@ -0,0 +1,42 @@
+#this file is used to generate the package-info.java class that
+# records the version, revision, branch, user, timestamp, and url
+unset LANG
+unset LC_CTYPE
+unset LC_TIME
+version=$1
+build_dir=$2
+user=`whoami`
+date=`date`
+cwd=`pwd`
+if [ -d ../.git ]; then
+  revision=`git log -1 --pretty=format:"%H"`
+  hostname=`hostname`
+  branch=`git branch | sed -n -e 's/^* //p'`
+  url=`git remote -v | sed -n -e 's/^origin[[:space:]]\(.*\)(fetch)/\1/p'`
+elif [ -d .svn ]; then
```

Comment:
@mnogu 
Thank you for your advice. 
But users can clone with Subversion. If so , I think  it's necessary to deal with .svn .